functions/src/types/JWTToken.ts

```diff
@@ -0,0 +1 @@
+export type JWTToken = string
```

Comment:
Maybe regroup types `JWTKey` and `JWTToken` in one file `functions/src/types/JWT.ts`. 

```suggestion
export type JWTToken = string
export type JWTKey = string
```